demo/demo-tekton

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+#Assumes User is logged in to cluster
+set -euo pipefail
+
+all_cmds_found=0
+for cmd in oc tkn; do
+   if ! which -s $cmd; then
```

Comment:
I am in favor of using 'type' then which. Also -s is not available in all 'which' versions.